ExampleApplication/TWTSampleViewController.m

```diff
@@ -52,6 +56,13 @@ - (void)viewDidLoad
     self.examples = [NSArray arrayWithContentsOfFile:toastSamplesURLString];
     
     [self.tableView registerClass:[UITableViewCell class] forCellReuseIdentifier:twt_cellIdentifier];
+
+    // TWTLocationController
+    // Update info.plist with either NSLocationWhenInUseUsageDescription and/or NSLocationAlwaysUsageDescription
+    [[TWTLocationController sharedInstance] fetchCurrentLocationWithCompletion:^(CLLocation *location, NSError *error) {
```

Comment:
I suggest that this not be just something in the Sample View Controller. This really should be it's own sample view. See how the other samples are done in there. Hint, look into ToastExamples.plist